CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 - Check new project names against Erlang standard library modules to avoid clashes.
 - Formatter changed to treat lines with only spaces as empty.
 - Improve clarity of HTML table display in documentation.
+- Fix use & display of gleam.toml links in documention.
```

Comment:
Could you remove this entry? Just an internal thing so no need to expose it to the users' changelog

Reply:
Sure